tests/StorageBackendTest.php

```diff
@@ -79,71 +79,6 @@ public function testStorageAssignRedshiftBackend()
         $this->client->removeProjectStorageBackend($project['id'], $redshiftBackend['id']);
     }
 
-    public function testStorageAssignSnowflakeBackend()
-    {
-        // get redshift backend
-        $backends = $this->client->listStorageBackend();
-        $snowflakeBackend = null;
-        foreach ($backends as $backend) {
-            if ($backend['backend'] == 'snowflake') {
-                $snowflakeBackend = $backend;
-                break;
-            }
-        }
-        if (!$snowflakeBackend) {
-            $this->fail('Snowflake backend not found');
-        }
-
-        $name = 'My org';
-        $organization = $this->client->createOrganization($this->testMaintainerId, [
-            'name' => $name,
-        ]);
-
-        $project = $this->client->createProject($organization['id'], [
-            'name' => 'My test',
-        ]);
-
-        $this->assertArrayHasKey('backends', $project);
-        $this->assertEquals('mysql', $project['defaultBackend']);
-
-        $backends = $project['backends'];
-
-        $this->assertArrayHasKey('mysql', $backends);
-        $this->assertArrayNotHasKey('snowflake', $backends);
-
-
-        $this->client->assignProjectStorageBackend($project['id'], $snowflakeBackend['id']);
-
-        $project = $this->client->getProject($project['id']);
-        $backends = $project['backends'];
-
-        $this->assertArrayHasKey('snowflake', $backends);
-        $this->assertEquals('snowflake', $project['defaultBackend']);
-
-        $redshift = $backends['snowflake'];
-        $this->assertEquals($snowflakeBackend['id'], $redshift['id']);
-
-        // let's try to create a bucket in project now
-
-        $token = $this->client->createProjectStorageToken($project['id'], [
-            'description' => 'test',
-            'expiresIn' => 60,
-            'canManageBuckets' => true,
-        ]);
-
-        $sapiClient = new \Keboola\StorageApi\Client([
-            'url' => getenv('KBC_MANAGE_API_URL'),
-            'token' => $token['token'],
-        ]);
-        $bucketId = $sapiClient->createBucket('test', 'in');
-        $bucket = $sapiClient->getBucket($bucketId);
-        $this->assertEquals('snowflake', $bucket['backend']);
-
-        $sapiClient->dropBucket($bucketId);
-
-        $this->client->removeProjectStorageBackend($project['id'], $snowflakeBackend['id']);
-    }
-
```

Comment:
I guess as default we won't be using this anymore, but what if we wanted to assign a foreign snowflake backend?

Reply:
it is no problem, it can be assigned to project.